addons/hr_recruitment/views/res_config_settings_views.xml

```diff
@@ -40,6 +40,18 @@
                                     <div class="content-group" id="interview_forms"/>
                                 </div>
                             </div>
+                            <div class="col-xs-12 col-md-6 o_setting_box" id="sms">
+                                <div class="o_setting_right_pane" id="sms_settings">
+                                    <div class="o_form_label">
+                                        Send SMS
+                                        <a href="https://www.odoo.com/documentation/16.0/applications/marketing/sms_marketing/pricing/pricing_and_faq.html" title="Documentation" class="ms-1 o_doc_link" target="_blank"></a>
+                                    </div>
+                                    <div class="text-muted">
+                                        Send texts to your contacts
+                                    </div>
+                                    <widget name="iap_buy_more_credits" service_name="sms" hide_service="1"/>
```

Comment:
AFAIK, this widget is the problem, and you are putting it again.

Reply:
Already fixed. I saw it right after pushing the branch. Thanks

Reply:
It's still there.

Reply:
Done